lib/collection.go

```diff
@@ -142,7 +166,7 @@ func (collectionImpl) List(scope scope, p *ListParams) ([]dsref.VersionInfo, err
 			infos[i] = reporef.ConvertToVersionInfo(&r)
 		}
 	} else if listProfile.Peername == "" || reqProfile.Peername == listProfile.Peername {
-		infos, err = base.ListDatasets(scope.Context(), scope.Repo(), p.Term, restrictPid, p.Offset, p.Limit, p.Public, p.ShowNumVersions)
+		infos, err = base.ListDatasets(scope.Context(), scope.Repo(), p.Term, restrictPid, p.Offset, p.Limit, p.Public, true)
```

Comment:
lines 123-191 here are fallback in case we don't have a collection.

Do we still need this? Or is the idea that we should really just be relying on the collection /list?

Reply:
It's inefficient for `base.ListDatasets` to get the number of versions, it has to traverse the `PreviousPath` field for each version. That's why it was made opt-in when calling `qri list --num-versions`. Could even see defaulting this value to `false` instead, as long as no api consumer depends on this behavior. It's fine either way, since this is hopefully going away soon as we move entirely over to collections.

Reply:
What's the deciding line that we have officially moved everything over to collections?

Reply:
Definitely not using dscache at all, and I think repo will go away once we're confident that collection is working as intended. To simplify my previous comment, I think this parameter should be `false` instead of `true`.